pillar.example

```diff
@@ -96,7 +96,7 @@ users:
     gitconfig:
       user.name: B User
       user.email: buser@example.com
-      url."https://".insteadOf: "git://"
+      "url.https://.insteadOf": "git://"
```

Comment:
Yes, I got it that the quotes are to make salt happy, only was not sure it would render OK. If you've tested it and it renders OK (as you pasted above), then I'm all OK with this. 😄